lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -97,6 +97,20 @@
                 "" // another entry for fun
             ]
         }
+        // no return value
+    }
+
+    let IsWellFormedLanguageTag = function (langTag) {
+        let retVal = platform.isWellFormedLanguageTag(langTag);
+        if (retVal === null) {
+            if (!LANG_TAG_RE) {
```

Comment:
you have this code pattern in multiple places. Consider moving it to a function and calling it.

Reply:
I think the end shape of each of these cases may be different enough that adding the overhead of another call for it isn't worthwhile. (I believe that Intl code is not inlined into user code but I don't know to what extent it is internally optimized during runtime.)

Reply:
I assumed you were referring to this pattern.

```
+    let IsWellFormedLanguageTag = function (langTag) {
+        let retVal = platform.isWellFormedLanguageTag(langTag);
```

I think you might also be referring to the check-and-call pattern around `InitializeLangTagREs();`. I figured that the check inline and then calling the function if necessary was cheaper than calling the function and doing the check inside (i.e. move the checks inside and rename `InitializeLangTagREs` to `EnsureLangTagREs`). For now, I'll stand by this pattern of checking outside the init function, but I'll reconsider refactoring this in the future.
 #Pending

Reply:
SGTM